manifests/coreos-bootimages.json

```diff
@@ -0,0 +1,16 @@
+{
+    "apiVersion": "v1",
+    "kind": "ConfigMap",
+    "metadata": {
+        "name": "coreos-bootimages",
+        "namespace": "openshift-machine-config-operator",
+        "annotations": {
+            "include.release.openshift.io/ibm-cloud-managed": "true",
+            "include.release.openshift.io/self-managed-high-availability": "true",
+            "include.release.openshift.io/single-node-developer": "true"
```

Comment:
Would you happen to have a reference for what these annotations signify?

Reply:
See
https://github.com/openshift/enhancements/blob/master/enhancements/update/cluster-profiles.md

Admittedly I cargo culted these from the MCO; we want this data in all of them I think.  It's not directly relevant for e.g. `single-node-developer` but it also doesn't do anything and it's tiny.